seqr/management/commands/check_for_new_samples_from_pipeline.py

```diff
@@ -111,8 +112,9 @@ def handle(self, *args, **options):
         session = AirtableSession(user=None, no_auth=True)
         for project, sample_ids in samples_by_project.items():
             project_sample_data = update_sample_data_by_project[project.id]
-            is_internal = notify_search_data_loaded(
-                project, dataset_type, sample_type, inactivated_sample_guids,
+            is_internal = not project_has_anvil(project) or is_internal_anvil_project(project)
```

Comment:
This was actually recommended on my split PDO PR but I opted against it because of the chnges to shared functionality. Since the function is no longer shared, I agree that it makes more sense to split it out here

Reply:
@jklugherz any feedback blocking your approval?